iceoryx2-bb/cxx/include/iox2/legacy/cli/arguments.hpp

```diff
@@ -1,16 +1,14 @@
 // Copyright (c) 2022 by Apex.AI Inc. All rights reserved.
+// Copyright (c) 2025 Contributors to the Eclipse Foundation
+//
+// See the NOTICE file(s) distributed with this work for additional
```

Comment:
@elfenpiff I noticed that there is not NOTICE file. I guess it it time to add one